content/docs/user-guide/experiment-management/cleaning-experiments.md

```diff
@@ -210,16 +210,16 @@ Removed experiments: exp-bb09c
 
 ## Removing queued experiments
 
-When you've created experiments to be run in the queue with
-`dvc exp run --queue` and later decide not to run them, you can remove them with
-`dvc exp remove --queue`.
+When you've created experiments to be run in the queue with `dvc queue start`
+and later decide not to run them, you can remove them with
+`dvc queue remove --queued`.
```

Comment:
I thought we agreed that `dvc exp remove --queue` will stay and we will generally recommend `exp` over low-level `queue` commands? Not that important, just surprised to see this suggestion.

Reply:
I rolled this back and rephrased a little (675947351) to preserve the original intention (explained by @pmrowla in https://github.com/iterative/dvc.org/pull/3894#discussion_r954598608). PTAL

@dberenbaum But it wasn't that clear to me that a) we won't recommend `queue start` over `exp run --all` at least until new aliases are decided; and b) why was `exp remove --queue` changed to `queue remove --queued` [in the original PR](https://github.com/iterative/dvc.org/pull/3715/files#diff-213c105f41cfb7e3abe29cba4fd80d9cf459bb94703b0a49708b23cf6b6cf5c5L171-R172) then? (Should we roll that back?)